x/evm/keeper/msg_server.go

```diff
@@ -1036,11 +907,9 @@ func (s msgServer) AddChain(c context.Context, req *types.AddChainRequest) (*typ
 		return nil, fmt.Errorf("TSS is disabled")
 	}
 
```

Comment:
On second thought, let's make `AddChain` msg require the access control role. Before, a compromised controller key could only add pending chains, but now it'll pollute the chain list even if it's not activated.

Reply:
agree, let's move to governance

Reply:
done